src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToStartHandlerTest.java

```diff
@@ -269,4 +273,52 @@ public void givenAppealWithNoHearingRecordingRequestOutstanding_doNotPopulateDir
         assertEquals(2, listOptions.size());
         assertEquals(expected, response.getData().getDirectionTypeDl());
     }
+
+    @Test
+    public void givenAValidCallbackType_thenClearTheConfidentialityFields() {
+        handler = new DirectionIssuedAboutToStartHandler();
+        sscsCaseData.setConfidentialityType(ConfidentialityType.CONFIDENTIAL.getCode());
+        sscsCaseData.setSendDirectionNoticeToFTA(YesNo.YES);
+        sscsCaseData.setSendDirectionNoticeToRepresentative(YesNo.YES);
+        sscsCaseData.setSendDirectionNoticeToOtherPartyRep(YesNo.YES);
+        sscsCaseData.setSendDirectionNoticeToOtherPartyAppointee(YesNo.YES);
+        sscsCaseData.setSendDirectionNoticeToOtherParty(YesNo.YES);
+        sscsCaseData.setSendDirectionNoticeToJointParty(YesNo.YES);
+        sscsCaseData.setSendDirectionNoticeToAppellantOrAppointee(YesNo.YES);
+
+        handler.handle(ABOUT_TO_START, callback, USER_AUTHORISATION);
+
+        assertNull(sscsCaseData.getConfidentialityType());
+        assertNull(sscsCaseData.getSendDirectionNoticeToFTA());
+        assertNull(sscsCaseData.getSendDirectionNoticeToRepresentative());
+        assertNull(sscsCaseData.getSendDirectionNoticeToOtherPartyRep());
+        assertNull(sscsCaseData.getSendDirectionNoticeToOtherPartyAppointee());
+        assertNull(sscsCaseData.getSendDirectionNoticeToOtherParty());
+        assertNull(sscsCaseData.getSendDirectionNoticeToJointParty());
+        assertNull(sscsCaseData.getSendDirectionNoticeToAppellantOrAppointee());
+    }
+
+    @Test
+    public void givenAValidCallbackType_thenVerifyAllPartiesOnTheCase() {
+        handler = new DirectionIssuedAboutToStartHandler();
+
+        CcdValue<OtherParty> otherParty = CcdValue.<OtherParty>builder()
+                .value(OtherParty.builder()
+                        .id("1")
+                        .name(Name.builder().firstName("Harry").lastName("Kane").build())
+                        .isAppointee(YES.getValue())
+                        .appointee(Appointee.builder().id("2").name(Name.builder().firstName("Henry").lastName("Smith").build()).build())
+                        .rep(Representative.builder().id("3").name(Name.builder().firstName("Wendy").lastName("Smith").build()).hasRepresentative(YES.getValue()).build())
+                        .build())
+                .build();
```

Comment:
I don't like this indentation. I was going to allow it until I saw hasRepresentative and I couldn't work out which builder that's a part of. When it takes longer than 2 seconds to recognise, then you know you can improve with correct indentation. Please do the proper separation with indents for each builder here. (Name.builders can arguably be on a single line as they are simple and obvious).